core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -17,17 +17,41 @@
 
 package kafka.tools
 
-import java.io.{PrintStream, FileOutputStream}
+import java.io.{FileOutputStream, PrintStream}
+import java.util.Properties
 
 import kafka.common.MessageFormatter
-import kafka.consumer.{BaseConsumer, BaseConsumerRecord}
-import kafka.utils.TestUtils
+import kafka.consumer.{BaseConsumer, BaseConsumerRecord, OldConsumer, Whitelist}
+import kafka.integration.KafkaServerTestHarness
+import kafka.server.KafkaConfig
+import kafka.utils.{TestUtils, ZkUtils}
 import org.easymock.EasyMock
 import org.junit.Assert._
 import org.junit.Test
-import org.scalatest.junit.JUnitSuite
 
-class ConsoleConsumerTest extends JUnitSuite {
+class ConsoleConsumerTest extends KafkaServerTestHarness {
```

Comment:
This change makes all the existing tests in this class _much_ slower. It would be better to create a separate test class (maybe `ConsoleConsumerIntegrationTest`) or to use mocks in this test.